awxkit/awxkit/api/pages/api.py

```diff
@@ -325,10 +335,12 @@ def _assign_related(self):
                         post_data = {'id': rel_page['id']}
                         endpoint.post(post_data)
                         log.error("endpoint: %s, id: %s", endpoint.endpoint, rel_page['id'])
+                        self.__has_error = True
```

Comment:
Dropping this comment here because the actual line is currently out of scope...

There is a line a few lines above here with a bare `FIXME` comment.  Not having a `log.error()` call there is the cause of the problem tracked in https://github.com/ansible/tower/issues/5889.  That line will eventually need a log call plus your error attribute, and it probably makes sense to just fold that change in here in your PR.

Reply:
what should the error message read?

log.error("failed to get natural key from cache: %s, id: %s", endpoint.endpoint, rel_page['id']) ?

Reply:
just had a thought - our integration tests in internal jenkins which have a collections test-set should pick up and possible errors with `import.py` in the collections

Reply:
> what should the error message read?
> 
> `log.error("failed to get natural key from cache: %s, id: %s", endpoint.endpoint, rel_page['id'])` ?

No, the error messaging around natural keys seems to be confusing for users, and in this case there is a more straightforward explanation -- there doesn't seem to be a matching object on this Tower instance for the imported relation.